incubator/redash/templates/_helpers.tpl

```diff
@@ -6,43 +6,99 @@ Expand the name of the chart.
 {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "redash.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
 {{/*
 Create a default fully qualified app name.
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
 */}}
 {{- define "redash.fullname" -}}
+{{- if .Values.fullnameOverride -}}
```

Comment:
If `fullnameOverride` is set, you get the same for `redash.fullname`, `redash.adhocWorker.fullname`, etc. I guess this is not intended and will cause problems/name clashes.